src/test/cypress/support/pageobjects/assessment/ModelingExerciseAssessmentEditor.ts

```diff
@@ -1,21 +1,25 @@
 import { AbstractExerciseAssessmentPage } from './AbstractExerciseAssessmentPage';
-import { MODELING_SPACE } from '../exercises/modeling/ModelingEditor';
+import { MODELING_EDITOR_CANVAS } from '../exercises/modeling/ModelingEditor';
 import { CypressExerciseType } from '../../requests/CourseManagementRequests';
 import { BASE_API, PUT } from '../../constants';
 // TODO: find or create better selectors for this
-const FEEDBACK_CONTAINER = '.sc-lcuiOb';
+const ASSESSMENT_CONTAINER = '#modeling-assessment-container';
 
 /**
  * A class which encapsulates UI selectors and actions for the Modeling Exercise Assessment editor
  */
 export class ModelingExerciseAssessmentEditor extends AbstractExerciseAssessmentPage {
     openAssessmentForComponent(componentNumber: number) {
-        cy.get('.apollon-row').getSettled(`${MODELING_SPACE} >> :nth-child(${componentNumber})`).children().eq(0).dblclick('top', { force: true });
+        cy.get('.apollon-row').getSettled(`${MODELING_EDITOR_CANVAS} >>> :nth-child(${componentNumber})`).children().eq(0).dblclick('top', { force: true });
```

Comment:
Was the `>>>` operator intentionally added here or is this a typo?

Reply:
Intentional as the id is one level lower than the selector we were previously using.